pixi/webpack.config.js

```diff
@@ -3,18 +3,26 @@ const webpack = require('webpack');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 const FileManagerPlugin = require('filemanager-webpack-plugin');
 const TerserPlugin = require('terser-webpack-plugin');
+const WebpackBuildNotifierPlugin = require('webpack-build-notifier');
 
 const config = require('./config.js');
 const {calculateHash} = require('@ampproject/toolbox-script-csp');
 
 module.exports = (env, argv) => {
   const mode = argv.mode || 'production';
+  const isDevelopment = mode == 'development';
 
   return {
     entry: path.join(__dirname, 'src/ui/PageExperience.js'),
     output: {
       filename: 'pixi.[name].[contenthash].js',
       chunkFilename: 'pixi.[name].[chunkhash].bundle.js',
```

Comment:
Shouldn't this be removed in favour of the next lines?

Reply:
Ugh...